wg-manifests/charter.md

```diff
@@ -1,70 +1,32 @@
-
-- Coordinate with distribution owners, to make sure they are in-sync about the
-  release schedule and have time to test and bring their distributions
-  up-to-date.
-
-### Out of scope
-
-This WG is NOT going to:
-- Maintain deployment-specific tools like `kfctl`.
-- Maintain distribution-specific manifests.
-- Decide which applications to include in Kubeflow.
-- Decide which variant of an application to include (e.g., KFP Standalone vs
-  KFP with Istio).
-- Create and maintain one or more Kubeflow distributions.
-- Support configurations with environment-specific requirements, like special
-  hardware, different versions of third-party apps (e.g., Istio, KNative, etc.)
-  or custom OIDC providers.
-- Support and promote a specific deployment tool (e.g., `kfctl`). Opinionated
-  deployment tools can extend the base kustomizations to create manifests that
-  support their methods.
-    - For example, people invested in `kfctl` can create overlays that enable
-      the use of `kfctl`'s parameter substitution, which expects a specific
-      folder structure (`params.env`).
+- Enable users / distributions to install, extend and maintain Kubeflow as a multi-tenant platform for multiple users
+- This includes dependencies, security efforts and exemplary integration with popular tools and frameworks.
+- Synchronize the manifests (Helm, Kustomize) between working groups
+- We try to be compatible with the popular Kubernetes clusters (Kind, Rancher, AKS, EKS, GKE, ...)
+- **We do not support a specific deployment tool (e.g., ArgoCD, Flux)**
+- The default installation shall not contain deep integration with external cloud services or closed source solutions, instead we aim for Kubernetes-native solutions and light authentication and authorization integration with external IDPs
+- We provide hints and experimental examples how a user / distribution could integrate non-default external authentication (e.g. companies Identity Provider) and popular non-default services on his own
+- There is the evolving and not exhaustive list of dependencies for a proper multi-tenant platform installation: Istio, KNative, Dex, Oauth2-proxy, Cert-Manager, ...
+- There is the evolving and not exhaustive list of applications:  KFP, Trainer, Dashboard, Workspaces / Noteboks, Kserve, Spark, ...
+
+## Communication Tasks
+
+### With Application Owners
+
+- Aid the application owner in creating manifests (Helm, Kustomize) for his application
+- Aid the application owner regarding security best practices
+- Communicate with the application owner regarding releases and versioning
+
+### With Users / Distribution Owners
+- Distributions are strongly opinionated derivatives of Kubeflow platform/manifests, for example replacing all databases with closed source managed databases from AWS, GKE, Azure, ...
+- A distribution can be created by an arbitrary amount of users / companies in private or in public by deriving from Kubeflow platform/manifests, see the definition above
```

Comment:
This is not the current definition of a distribution, there is no requirement for it to be "strongly opinionated".

If we are to include a definition here, we must agree as a community.

It might be better to just define it somewhere else rather than in this charter.

Reply:
Well I can remove the word "strongly" but i think that is right where it belongs.

Reply:
Regardless of where we define the term "Kubeflow Distribution" is defined, changing it will require a wider community discussion, because we already have a definition in the original KEP.

https://github.com/kubeflow/community/blob/master/proposals/434-kubeflow-distribution/README.md